apps/sms/test/marionette/conversation_test.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 var assert = require('chai').assert;
-
+var ThreadMock = require('./mocks/mock_message_thread');
```

Comment:
I'm wondering maybe having this thread generator somewhere in ```libs/threads.js (maybe?)``` would be more appropriate? Initially we thought that ```mocks``` folder is only for API mocks (well we have storage mock there, but its just backing API mocks).

IMO, Thread generator is more related to test logic: ```what data we want to test in this particular test/suite?```

What do you think?